src/App.tsx

```diff
@@ -1,24 +1,174 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable */
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos, addNewTodo, deleteTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import { Error } from './utils/Error';
 
-const USER_ID = 0;
+const USER_ID = 6752;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterValue, setFilterValue] = useState('all');
+  const [errorMessage, setErrorMessage] = useState<Error>(Error.NoError);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [showNotification, setShowNotification] = useState<boolean>(false);
```

Comment:
delete this state, and check if errorMessage === Error.NoError, and when you need to clear state use setErrorMessage(Error.NoError)